linux_os/guide/services/ntp/chronyd_no_chronyc_network/rule.yml

```diff
@@ -0,0 +1,30 @@
+documentation_complete: true
+
+prodtype: rhel8,fedora,ol8
```

Comment:
Is this truly only applicable to RHEL 8?

Reply:
@yuumasato, similar question to the above.